.github/workflows/sort.yml

```
@@ -0,0 +1,17 @@
+on: [push, workflow_dispatch]
+
+jobs:
+  sort:
+    runs-on: ubuntu-latest
+    name: Sort Data
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: "14"
```

Does it also work with node version 16 or 17?
```suggestion
          node-version: "16"
```

---

16 is LTS now, by the way.